client/src/components/History/CurrentHistory/HistoryFilters.vue

```diff
@@ -27,32 +27,49 @@
             </b-input-group-append>
         </b-input-group>
         <div v-if="showAdvanced" class="mt-2" description="advanced filters">
```

Comment:
Can we add the `@keyup.enter="onSearch"` here, instead of adding another child component.

Reply:
My bad i didn't think of that, yes it works too.

Reply:
On that note, how about also adding `keyup.esc` for toggling out of the advanced menu as well?

Reply:
Yes, that sounds good to me.